client/landing/gutenboarding/hooks/use-on-site-creation.ts

```diff
@@ -54,52 +50,30 @@ export default function useOnSiteCreation(): void {
 			setIsRedirecting( true );
 
 			if ( selectedPlan && ! selectedPlan?.isFree && planProductSource ) {
-				const planProduct: RequestCartProduct = createRequestCartProduct( {
-					product_id: planProductSource.productId,
-					product_slug: planProductSource.storeSlug,
-					extra: {
-						source: 'gutenboarding',
-					},
-				} );
+				const productSlugsToAddToCart: string[] = [ planProductSource.storeSlug ];
 
-				let domainProduct: RequestCartProduct | null = null;
 				if ( domain?.product_id && domain?.product_slug ) {
-					domainProduct = createRequestCartProduct( {
-						meta: domain.domain_name,
-						product_id: domain.product_id,
-						product_slug: domain.product_slug,
-						extra: {
-							privacy: domain.supports_privacy,
-							source: 'gutenboarding',
```

Comment:
I like this approach a lot, however I wonder if removing this prop has other consequences. Is `source` used on the backend for e.g. analytics?

Reply:
Maybe, but as Gutenboarding is going to be retired, I feel that the tradeoff is _probably_ worth it?

Reply:
If this is a blocker, we could add a new query string to the URL to set this explicitly.

Reply:
I added a new query param to preserve this property in 638eeb4183